### DISCUSSION TITLE
Canopy with the free tier

### DISCUSSION THREAD

**Discussion by jicowan** [pinecone-io/canopy#316] [General]
Is it possible to use Canopy with the free tier of Pinecone or do you need to upgrade to a paid account? When I run `canopy new` I get a message that says `Index creation failed. To create serverless indexes, you must upgrade your plan.` I only saw 2 options in the default config file: serverless or pod.

**Comment by tiago-falves** [comment]
Did you solve this?

**Reply by jicowan** [reply]
Yes, thanks.

**Reply by tiago-falves** [reply]
> Yes, thanks.

Actually I am having the same problem! Could you please share your solution for creating the server less index with canopy free tier? 
Thank you so much!
